notebooks/PyRadiomicsExample.ipynb

```diff
@@ -367,33 +369,33 @@
       "\t original_firstorder_10Percentile : 632.0\n",
       "\t original_firstorder_Kurtosis : 2.18077293939\n",
       "\t original_firstorder_Mean : 825.235436307\n",
-      "\t original_glcm_SumVariance : 103.142793792\n",
-      "\t original_glcm_Homogeneity1 : 0.276140402104\n",
-      "\t original_glcm_Homogeneity2 : 0.189156155892\n",
-      "\t original_glcm_ClusterShade : -52.9707943386\n",
-      "\t original_glcm_MaximumProbability : 0.00792784235012\n",
-      "\t original_glcm_Idmn : 0.957796447609\n",
-      "\t original_glcm_Contrast : 52.2310659277\n",
-      "\t original_glcm_DifferenceEntropy : 3.79686113536\n",
-      "\t original_glcm_InverseVariance : 0.188666637795\n",
-      "\t original_glcm_Dissimilarity : 5.58932678922\n",
-      "\t original_glcm_SumAverage : 33.4497492152\n",
-      "\t original_glcm_DifferenceVariance : 17.6107741076\n",
-      "\t original_glcm_Idn : 0.866370546902\n",
-      "\t original_glcm_Idm : 0.189156155892\n",
-      "\t original_glcm_Correlation : 0.335214788202\n",
-      "\t original_glcm_Autocorrelation : 292.684050471\n",
-      "\t original_glcm_SumEntropy : 5.31547876648\n",
-      "\t original_glcm_AverageIntensity : 17.1242601309\n",
-      "\t original_glcm_Energy : 0.00290880217681\n",
-      "\t original_glcm_SumSquares : 39.9781084143\n",
-      "\t original_glcm_ClusterProminence : 26251.1709801\n",
-      "\t original_glcm_Entropy : 8.79428086119\n",
-      "\t original_glcm_Imc2 : 0.692033706271\n",
-      "\t original_glcm_Imc1 : -0.091940840043\n",
-      "\t original_glcm_DifferenceAverage : 5.58932678922\n",
-      "\t original_glcm_Id : 0.276140402104\n",
-      "\t original_glcm_ClusterTendency : 103.142793792\n",
+      "\t original_glcm_SumVariance : 108.731393255\n",
```

Comment:
It is not obvious from the title and comments in the PR that it should lead to a change in the feature baselines. Is this expected?